.gitignore

```diff
@@ -38,3 +38,6 @@ docker/data/
 
 # To avoid crazy stuff on some PR, we must manually FORCE ADD IT on each new release
 composer.lock
+
+# To avoid IDE specific files
+.idea/*
```

Comment:
Can you remove these lines please? 

It's better to put them in your global gitignore.